include/wx/generic/grid.h

```diff
@@ -2816,6 +2825,11 @@ class WXDLLIMPEXP_CORE wxGrid : public wxScrolledCanvas
     // setting m_isDragging to true
     wxPoint m_startDragPos;
 
+public:
+    // the position of the last mouse event
+    // used for detection of the movement direction
+    wxPoint m_lastMousePos;
```

Comment:
I think this needs to be reset somewhere when the operation is over, but currently this doesn't seem to be done, could you please check it?